README.md

```diff
@@ -39,15 +39,18 @@
 By using this software you agree to the terms in COPYING
 
 ## Installation:
- - Install TensorFlow(>1.1), otherwise TensorMol is self-contained.
+ - Install TensorFlow(>1.1) as well as scipy, otherwise TensorMol is self-contained.
```

Comment:
We should defer to `requirements.txt` and setup.py https://github.com/Exabyte-io/TensorMol/pull/2/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R44 instead

Reply:
requirements.txt is gone, readme is updated to full-send  the `pip install -e .` route